lib/src/common/variables.js

```diff
@@ -167,6 +169,7 @@ export const globalTokens = {
   // Padding
   padding_08: "8px",
   padding_16: "16px",
+  padding_24: "24px",
```

Comment:
Why are we using these padding variables having a spacing scale? Is it because the (spacing) naming not being as useful as these ones? (referencing the value directly) or for other reasons?

* `padding_16` = `spacing_03`
* `padding_16` = `spacing_05`
* `padding_24` = `spacing_06`

Reply:
Seems to be no apparent reason, so I will remove padding_08, padding_16 and padding_24 variables and use the spacing ones.
Thanks Álex!